website/src/blogs/2018-01-07-install-openebs-using-stackpointcloud-trusted-charts.md

```diff
@@ -0,0 +1,148 @@
+- [Kubeturbo](https://github.com/turbonomic/kubeturbo)
+- [Memcached](https://memcached.org/)
+- [Tensorflow Inception](https://github.com/tensorflow/models/tree/master/research/inception)
+
+I’ll go through the quick steps of deploying OpenEBS.
+
+### Prerequisites
+
+Minimum requirements for deploying your Kubernetes clusters on StackPointCloud:
+
+### Cloud Provider Account
+
+- [Amazon Web Services (AWS)](https://aws.amazon.com/) or
+- [DigitalOcean](https://www.digitalocean.com)
+
+### Deploy a New Kubernetes Cluster
+
+First, go to [stackpoint.io](https://stackpoint.io/) and click on **Launch a Cluster** button to start your free trial.
+
+![Universal control plane for managed Kubernetes](https://cdn-images-1.medium.com/max/800/0*0cB3ttYmslFZgH1h.png)
+
+Then choose your cloud provider. In this example, I will use **Digital Ocean**.
+
+![Choose cloud provider](https://cdn-images-1.medium.com/max/800/0*21G24JgfuqlR6snZ.png)
+
+### Configure Access to Digital Ocean
+
+On the next screen, we need to configure our provider. You need to provide Digital Ocean API Token and optionally your SSH Key.
+
+![Configure your provider](https://cdn-images-1.medium.com/max/800/0*wDcMg-_HTjIOFvgb.png)
+
+Click on **Add API Token** button.
+
+![Add API token](https://cdn-images-1.medium.com/max/800/0*53wGtQ7eUt18u6pS.png)
+
+After you add your credentials, click on **Submit**.
+
+### Configure K8s Cluster
+
+On “Configure your cluster” page click the edit button on **Distribution** and choose **Ubuntu 16.04 LTS**.
+
+![Configure your cluster](https://cdn-images-1.medium.com/max/800/0*NvtnryAA8GNi-fyN.png)
+
+Change the **Cluster Name** something meaningful like **OpenEBS Demo**.
+
+![Enter cluster name](https://cdn-images-1.medium.com/max/800/0*LTa6zBooJdTsyqss.png)
+
+Leave everything else as default and click on **Submit**.
+
+In about 10–15 minutes you will get your new cluster deployed.
+
+### Adding OpenEBS to Your Kubernetes Cluster
+
+First, make sure your cluster and all nodes are up.
+
+On the **Control Plane** tab click on your recently created cluster.
+
+![Control plane](https://cdn-images-1.medium.com/max/800/0*RHQ9LbyxydjHkJSk.png)
+
+Once the Kubernetes cluster is up on Digital Ocean with functional Helm, scroll down to the **Solutions** tab and click on **Add Solution** button.
+
+![Add Solution](https://cdn-images-1.medium.com/max/800/0*sH0lzv23vHonV5Zk.png)
+
+Click on **Add Solutions**, and select **Trusted Charts**.
+
+![Select charts](https://cdn-images-1.medium.com/max/800/0*V6iP5PzNAzFk4sME.png)
+
+From the list above select **OpenEBS**.
+
+![OpenEBS namespace](https://cdn-images-1.medium.com/max/800/0*CJkPrkJCS9Fp_GXu.png)
+
+**Release Name** is randomly generated every time. If you want to use OpenEBS example workloads provided in OpenEBS repos without any modification then use `default`as **NameSpace**. Otherwise, you need to modify the namespace for workloads you deploy and make sure to use the same name.
```

Comment:
```suggestion
**Release Name** is randomly generated every time. If you want to use OpenEBS example workloads provided in OpenEBS repos without any modification then use `default` as **NameSpace**. Otherwise, you need to modify the namespace for workloads you deploy and make sure to use the same name.
```